corehq/form_processor/backends/couch/update_strategy.py

```diff
@@ -20,6 +21,20 @@
 from dimagi.ext.couchdbkit import StringProperty
 
 
+def _coerce_to_datetime(v):
```

Comment:
does this return `None` if `v` is already a `datetime`? I would potentially either assert that you are only calling this on `date` items, or return the input if it is already a `datetime`.